examples/practice-typescript/src/components/TableController/index.tsx

```diff
@@ -0,0 +1,18 @@
+import Button from 'components/Button';
+import InputField from 'components/Input';
```

Comment:
Component name should same as file `Input`

Reply:
Fixed